content/docs/operating/configuration.md

```diff
@@ -268,6 +272,48 @@ Consul setups, the relevant address is in `__meta_consul_service_address`.
 In those cases, you can use the [relabel](#target-relabeling-relabel_config)
 feature to replace the special `__address__` label.
 
+### Kubernetes SD configurations `<kubernetes_sd_config>`
+
+CAUTION: Kubernetes SD is in beta: breaking changes to configuration are possible
+in future releases.
+
+Kubernetes SD configurations allow retrieving scrape targets from
+[Kubernetes'](http://kubernetes.io/) REST API. By default, this discovers nodes
+& appropriately annotated services so that metrics from both cluster
+components and deployed applications can be scraped. This will create multiple
+target groups: one for all nodes with each node as a target; and one for each
```

Comment:
Just checking, is this correct? "all nodes with each node". Don't know Kubernetes well enough to say, but do nodes contain other nodes?